src/utils/decorators.jsx

```diff
@@ -3,9 +3,21 @@ import { IntlProvider } from 'react-intl';
 
 export const decorateWithLocale = story => {
 	const locale = 'en-US';
+
+	/*
+	 * -- Inline SVG icon sprite --
+	 *
+	 * raw SVG sprite from `swarm-icons`
+	 */
+	const iconSpriteStyle = { display: 'none' };
+	const iconSprite = require('raw-loader!swarm-icons/dist/sprite/sprite.inc');
+
 	return (
 		<IntlProvider locale={locale}>
-			{story()}
+			<div>
+				<div style={iconSpriteStyle} dangerouslySetInnerHTML={{__html: iconSprite}} />
```

Comment:
the `<div>` is the single child, so that's fine. This uses `dangerouslySetInnerHTML` because the `iconSprite` is an HTML string fragment in a separate file - not a React component - so [this is how to load it](https://github.com/meetup/mup-web/blob/28bb929fa5f92bec244066304b77d64750c4fe77/src/components/PageWrap.jsx#L90)